server-common/src/test/java/org/apache/kafka/server/util/TopicFilterTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.server.util;
+
+import org.apache.kafka.common.internals.Topic;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+public class TopicFilterTest {
+
+    @Test
+    public void testIncludeLists() {
+
+        IncludeList topicFilter1 = new IncludeList("yes1,yes2");
+        assertTrue(topicFilter1.isTopicAllowed("yes2", true));
+        assertTrue(topicFilter1.isTopicAllowed("yes2", false));
+        assertFalse(topicFilter1.isTopicAllowed("no1", true));
+        assertFalse(topicFilter1.isTopicAllowed("no1", false));
+
+        IncludeList topicFilter2 = new IncludeList(".+");
+        assertTrue(topicFilter2.isTopicAllowed("alltopics", true));
+        assertFalse(topicFilter2.isTopicAllowed(Topic.GROUP_METADATA_TOPIC_NAME, true));
```

Comment:
What about the other internal topic? You can use the INTERNAL_TOPICS set and test all of them.

Reply:
`INTERNAL_TOPICS` variables is private so I didn't use it directly. But I added the test for the other internal topic.